downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -445,11 +414,44 @@ public function main() returns error? {
 }
 ```
 
+#### `persist` package
```

Comment:
Should these be reorganized to follow alphabetic order? Or is that only for connectors?

Reply:
@shafreenAnfar @NipunaRanasinghe @ThisaruGuruge @niveathika WDYT?

Reply:
is there any issue going with alphabetical order for the library module updates as well?

Reply:
I prefer alphabetical order because an unordered list might make it seem like the order is based on importance or impact, which isn't the case.